internal/controller/access.go

```diff
@@ -0,0 +1,108 @@
+package controller
+
+import (
+	"github.com/kintohub/kinto-cli/internal/api"
+	"github.com/kintohub/kinto-cli/internal/config"
+	"github.com/kintohub/kinto-cli/internal/utils"
+)
+
+func (c *Controller) Access() {
+
+	utils.CheckLogin()
+	utils.StartSpinner()
+
+	var envDetails []api.EnvDetails
+
+	envDetails = c.GetAvailableEnvNames(true)
+
+	utils.StopSpinner()
+	selectedEnvId, clusterId := SelectionPrompt(envDetails)
+	utils.StartSpinner()
+	var blocksToForward []api.RemoteConfig
+	blocksToForward = c.GetBlocksToForward(selectedEnvId)
+
+	if len(blocksToForward) != 0 {
+		utils.StopSpinner()
+		c.api.StartAccess(blocksToForward, selectedEnvId, clusterId)
+
+	} else {
+		utils.WarningMessage("No service/s found in this environment to teleport into!")
+	}
+
+}
+
+func (c *Controller) EnvironmentAccess(envId ...string) {
```

Comment:
Can u add docs to this func and maybe rename it to something that it does? I don't understand what this func does from `EnvironmentAccess`